CLI.md

```diff
@@ -100,7 +100,7 @@ A Byzantine-fault tolerant sidechain with low-latency finality and high throughp
 * `--recv-timeout-ms <RECV_TIMEOUT>` — Timeout for receiving responses (milliseconds)
 
   Default value: `4000`
-* `--max-pending-message-bundles <MAX_PENDING_MESSAGE_BUNDLES>` — The maximum number of incoming message bundles to include in a block proposal
```

Comment:
Is this an accidental regression?

Reply:
Ah good catch. I think I may need a rebase.

Reply:
@christos-h: You can run:

```bash
cargo run --locked --bin linera help-markdown > CLI.md
```

(That will probably just revert the file, but it makes sure it's in sync with the code.)